backend/src/test/java/gov/cdc/usds/simplereport/test_util/TestIdentityConfiguration.java

```diff
@@ -0,0 +1,51 @@
+package gov.cdc.usds.simplereport.test_util;
+
+import gov.cdc.usds.simplereport.config.authorization.DemoUserIdentitySupplier;
+import gov.cdc.usds.simplereport.config.simplereport.DemoUserConfiguration.DemoUser;
+import gov.cdc.usds.simplereport.service.model.IdentitySupplier;
+import java.util.List;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Primary;
+import org.springframework.security.authentication.TestingAuthenticationToken;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContext;
+import org.springframework.security.core.context.SecurityContextHolder;
+
+public class TestIdentityConfiguration {
```

Comment:
this has an identical IdentitySupplier to SliceTestConfiguration.java. Why not just keep that file, add the static methods to that file, and remove this one? seems like unnecessary decentralization.

Reply:
In the short term, because that file does a bunch of other stuff that we don't want in the full-stack test context. We could turn it around the other way and have that file import this one, but the next thing I'm going to do is make this file handle authorization for full-stack tests as well, which we might or might not want to do for slice tests.

Reply:
Noting for posterity: in the long term, this file may well not have a reason to exist, though in the slightly longer term it may again, depending how complicated the facilities situation gets. In the short term it is probably not worth explaining all of that, though perhaps the bare minimum would be wise in case the medium and long term get unfortunately deferred.